src/main/webapp/assets/modules/salary-report/sr-about-your-job.js

```diff
@@ -15,8 +15,33 @@ techlooper.directive("srAboutYourJob", function ($http, validatorService) {
           });
       }
 
+      var demandSkillSuggestion = function () {
+        delete scope.state.demandSkills;
+
+        var request = {};
+        scope.salaryReview.jobTitle && (request.jobTitle = scope.salaryReview.jobTitle);
+        scope.salaryReview.jobLevelIds && (request.jobLevelIds = vnwConfigService.getJobLevelIds(scope.salaryReview.jobLevelIds));
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.